src/GitHub.TeamFoundation.14/Services/VSGitServices.cs

```diff
@@ -86,6 +86,18 @@ public string GetLocalClonePathFromGitProvider()
             NavigateToHomePage(teamExplorer); // Show progress on Team Explorer - Home
             await WaitForCloneOnHomePageAsync(teamExplorer);
 #elif TEAMEXPLORER15 || TEAMEXPLORER16
+            // IGitActionsExt is proffered by SccProviderPackage, but isn't advertised.
+            // To ensure that getting IGitActionsExt doesn't return null, we first request the
+            // SccService which is advertised. This forces SccProviderPackage to load
+            // and proffer IGitActionsExt.
+            var gitSccServiceGuid = new Guid("28C35EB2-67EA-4C5F-B49D-DACF73A66989");
```

Comment:
I've confirmed with a Team Explorer developer that they have the wrong guid for this API.

A workaround would be to query for IGitExt instead which would force load the package and make IGitActionsExt available for you to use.

Please try that and let me know how it goes.

Reply:
I've changed it use `IGitExt` instead of the internal `SccService`. This is much simpler and does work. 👍